src/App.vue

```diff
@@ -163,15 +164,48 @@ import MiniWidgetContainer from './components/MiniWidgetContainer.vue'
 import SlideToConfirm from './components/SlideToConfirm.vue'
 import Alerter from './components/widgets/Alerter.vue'
 import { datalogger } from './libs/sensors-logging'
+import { useMainVehicleStore } from './stores/mainVehicle'
 import { useWidgetManagerStore } from './stores/widgetManager'
 
 const widgetStore = useWidgetManagerStore()
+const vehicleStore = useMainVehicleStore()
 
 const showConfigurationMenu = ref(false)
 
 // Main menu
 const showMainMenu = ref(false)
 
+let requestDisarmConfirmationPopup = false
+const openMainMenu = (): void => {
+  if (vehicleStore.isArmed) {
```

Comment:
You can do

```typescript
if (!vehicleStore.isArmed) {
 showMainMenu.value = true
}

...(logic for when it's armed)...
```

This way you remove a level of identation for most of the lines and avoid the extra `else`.